main.go

```diff
@@ -77,29 +77,49 @@ func (h handler) step1Insert(unit unit) (err error) {
 	return
 }
 
-// New setups the configuration assuming various parameters have been setup in the AWS account
-func New() (h handler, err error) {
+// NewDbConnexion setups the configuration assuming various parameters have been setup in the AWS account
+// TO DO: REVIEW THIS A LOT OF HARDCODED VALUES HERE...
+func NewDbConnexion() (h handler, err error) {
 
 	cfg, err := external.LoadDefaultAWSConfig(external.WithSharedConfigProfile("uneet-dev"))
```

Comment:
@kaihendry I'm almost done with this PR.
One thing I don't understand is why you need these here? What is the objective of doing that?